server/json/user1-institutions.json

```diff
@@ -6,47 +6,53 @@
       "status": {
         "code": 0,
         "message": ""
-      }
+      },
+      "currentSubmission": 0
```

Comment:
Yeah, but the reason this is here is so that the API doesn't have to be hit for every institution to check what the submission status is. We certainly *could* do that, but having the current submission associated with the institution is a convenience that will reduce backend load/front-end render times without (imo) mucking up the data model.